shared/store/action-logger.js

```diff
@@ -51,37 +15,17 @@ function makeActionToLog (action, oldState) {
   return noPayloadTransformer(action, oldState)
 }
 
-export const actionLogger = (store: any) => (next: any) => (action: any) => {
-  const shouldRunActionStats = shouldRunStats(actionStatFrequency)
-  const shouldRunLogStats = shouldRunStats(logStatFrequency)
+const logger = enableActionLogging ? setupLogger('actionLogger', 100, immediateStateLogging, immutableToJS, 50) : {log: () => {}}
 
+export const actionLogger = (store: any) => (next: any) => (action: any) => {
   const oldState = store.getState()
-
   const actionToLog = makeActionToLog(action, oldState)
+
   const log1 = [`Dispatching action: ${action.type}: `, forwardLogs ? JSON.stringify(actionToLog) : actionToLog]
+  console.log(log1)
```

Comment:
I'm a little confused by this. Shouldn't we be using `logger.log` here, and not the state below?